customagg/functions.py

```diff
@@ -0,0 +1,46 @@
+import re
+import logging
+import datetime as dt
+import math
+import numpy as np
+import pandas as pd
+
+from iotfunctions.base import BaseSimpleAggregator
+from iotfunctions.ui import (UIMultiItem,UIExpression)
+
+logger = logging.getLogger(__name__)
+
+# Specify the URL to your package here.
+# This URL must be accessible via pip install
+
+PACKAGE_URL = 'git+https://github.com/ankit-jha/addCustomIotFn@starter_agg_package'
```

Comment:
Do not keep your GIT URL in this custom function. 

Keep it generic:
PACKAGE_URL = 'git+https://XXXXXX@github.com/<user_id><path_to_repository>@starter_package'

Reply:
Done